src/javascript/app/pages/user/set_currency.js

```diff
@@ -276,13 +276,27 @@ const SetCurrency = (() => {
                 }
             });
         } else {
-            if ($submit) {
-                $submit.removeClass('button-disabled');
-            }
+            removeError(null, true);
             $error.text(localize('Please choose a currency')).setVisibility(1);
         }
     };
 
+    /**
+     * Remove error text if $error is defined
+     * Enable confirm button if is_btn_enabled is true
+     *
+     * @param {object} $error // error text jquery element
+     * @param {boolean} is_btn_enabled // Enable button
+     */
+    const removeError = ($error, is_btn_enabled)=>{
```

Comment:
please add the same spacing that we generally do
```suggestion
    const removeError = ($error, is_btn_enabled) => {
```